source/indexes/single-field-index.txt

```diff
@@ -0,0 +1,135 @@
+.. _ruby-single-field-index:
+
+====================
+Single Field Indexes
+====================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+ 
+.. meta::
+   :keywords: index, query, optimization, efficiency
+
+Overview
+--------
+
+Single field indexes are indexes with a reference to a single field of a
+document in a collection. These indexes improve single field query and
+sort performance. They also support :manual:`TTL Indexes </core/index-ttl>`
+that automatically remove documents from a collection after a certain
+amount of time or at a specified clock time.
+
+When creating a single field index, you must specify the following
+details:
+
+- The field on which to create the index
+- The sort order for the indexed values as either ascending or
+  descending
+
+.. note::
+
+   The default ``_id_`` index is an example of a single field index.
+   This index is automatically created on the ``_id`` field when a new
+   collection is created.
+
+Sample Data
+~~~~~~~~~~~
+
+The examples in this guide use the ``movies`` collection in the
+``sample_mflix`` database from the :atlas:`Atlas sample datasets
+</sample-data>`. To access this collection from your {+language+} 
+application, create a ``Mongo::Client`` object that connects to
+an Atlas cluster and assign the following values to your ``database`` 
+and ``collection``
+variables: 
+
+.. literalinclude:: /includes/indexes/single-field.rb
+   :start-after: start-sample-data
+   :end-before: end-sample-data
+   :language: ruby
+   :copyable:
+
+To learn how to create a free MongoDB Atlas cluster and
+load the sample datasets, see the :atlas:`Get Started with Atlas
+</getting-started>` guide.
+
+Create a Single Field Index
+---------------------------
+
+Use the ``create_one`` method to create a single
+field index. The following example creates an index in ascending order on the
+``title`` field:
+
+.. literalinclude:: /includes/indexes/single-field.rb
+   :start-after: start-index-single
+   :end-before: end-index-single
+   :language: ruby
+   :copyable:
+
+Verify the Index's Creation
```

Comment:
[s] Can potentially update this to [avoid using an article](https://www.mongodb.com/docs/meta/style-guide/style/titles-and-headings/#style-and-structure) and be more concise:
```suggestion
Verify Index Creation
```